deltacat/compute/compactor/compaction_session.py

```diff
@@ -71,7 +106,7 @@ def compact_partition(
         min_hash_bucket_chunk_size: int = 0,
         compacted_file_content_type: ContentType = ContentType.PARQUET,
         delete_prev_primary_key_index: bool = False,
-        read_round_completion: bool = False,
+        read_round_completion: bool = True,
```

Comment:
changing defaults is often backward incompatible. I'd recommend document motivation for the defaults as a doc.

Reply:
Agree with more document about default values. 
For `read_round_completion`, should be set to `True` because we need to enable multi-round compaction to limit input deltas when the partition is large. I think this flag set to `False` in `scaling-test` branch initially cause we don't want to read from previous test result which should be set back when merging back to `main`.